### ISSUE TITLE
segement crash

### ISSUE THREAD

**Issue by gxxzs** [strands-project/v4r_ros_wrappers#44]
when i click the segment botton, the project crash down
*** Error in `RTMT': corrupted size vs. prev_size: 0x00007f2148001de0 ***
Aborted (core dumped)
I install the both v4r and the wrappers from the Debian packages as shown in the instructions

**Comment by ToMadoRe** [comment]
just wanted to let you know that this repository is deprecated. You can find the latest public release on https://rgit.acin.tuwien.ac.at/v4r/v4r_ros_wrappers. Unfortunately though I do not know if that solves your problem.

**Comment by gxxzs** [comment]
> just wanted to let you know that this repository is deprecated. You can find the latest public release on https://rgit.acin.tuwien.ac.at/v4r/v4r_ros_wrappers. Unfortunately though I do not know if that solves your problem.

i have tried build v4r from source following the tips on https://rgit.acin.tuwien.ac.at/v4r/v4r, but when I run "make" the error happens because downloading...
       src='https://repo.acin.tuwien.ac.at/tmp/permanent/SiftGPU-V400.zip'
       dst='/home/top/v4r/3rdparty/SiftGPU/src/SiftGPU-V400.zip'
failure.
can i change this URL to some available website in china?

**Comment by ToMadoRe** [comment]
this file should be available directly in the repo (https://rgit.acin.tuwien.ac.at/v4r/v4r/blob/master/SiftGPU-V400.zip) via git lfs. You can try `git lfs pull`

**Comment by gxxzs** [comment]
> this file should be available directly in the repo (https://rgit.acin.tuwien.ac.at/v4r/v4r/blob/master/SiftGPU-V400.zip) via git lfs. You can try `git lfs pull`

I replace the URL as https://rgit.acin.tuwien.ac.at/v4r/v4r/blob/master/SiftGPU-V400.zip in the camkelist. Download fails too as HASH mismatch. which place in the cmakelist should i modify if  download it directly and put it under the folder SIftGpu/src. Thank you!

**Comment by ToMadoRe** [comment]
I don't think you need to modify https://rgit.acin.tuwien.ac.at/v4r/v4r/blob/master/cmake/3rdparty/BuildSiftGPU.cmake. 

Please just double-check that the SHA1 in your cmake file is the same as the one of your downloaded file https://rgit.acin.tuwien.ac.at/v4r/v4r/blob/master/SiftGPU-V400.zip
```
sha1sum SiftGPU-V400.zip
```
I just checked as well and for me the SHA1 sum of https://rgit.acin.tuwien.ac.at/v4r/v4r/blob/master/SiftGPU-V400.zip really is `eac08305f14cb35d12a22d493f84f9c21f6b7fdb`

**Comment by gxxzs** [comment]
> I don't think you need to modify https://rgit.acin.tuwien.ac.at/v4r/v4r/blob/master/cmake/3rdparty/BuildSiftGPU.cmake.
> 
> Please just double-check that the SHA1 in your cmake file is the same as the one of your downloaded file https://rgit.acin.tuwien.ac.at/v4r/v4r/blob/master/SiftGPU-V400.zip
> 
> ```
> sha1sum SiftGPU-V400.zip
> ```
> 
> I just checked as well and for me the SHA1 sum of https://rgit.acin.tuwien.ac.at/v4r/v4r/blob/master/SiftGPU-V400.zip really is `eac08305f14cb35d12a22d493f84f9c21f6b7fdb`

Thanks a lot. I download the SiftGPU-V400.zip and modify the cmake file. Then the make can go through. But there is one mistake that i cannot find a way to solve it. 

/usr/local/include/pcl-1.7/pcl/registration/eigen.h:49:41: fatal error: unsupported/Eigen/Polynomials: No such file or directory

I have change the pcl version to plc-1.8 which was build from source. But the mistake still remains.
